todo/mail/consumers/tracker.py

```diff
@@ -65,7 +65,7 @@ def parse_references(task_list, references):
     if answer_thread is None:
         logger.info("no answer thread found in references")
     else:
-        logger.info("found an answer thread: %d", answer_thread)
+        logger.info("found an answer thread: %s", str(answer_thread))
```

Comment:
Surprised this hadn't been caught before! I should get a test runner working on github. Thank you.